provisioner/roles/control_node/tasks/tower.yml

```diff
@@ -114,3 +114,23 @@
     body_format: json
     validate_certs: false
     force_basic_auth: true
+  when: not use_manifest
+
+- block:
+  - name: Load manifest into variable
+    local_action:
+      module: slurp
+      src: "{{ playbook_dir }}/manifest.zip"
+    register: manifest_file
+
+  - name: Post manifest file
+    uri:
+      url: https://{{ansible_host}}/api/v2/config/
+      method: POST
+      user: admin
+      password: "{{admin_password}}"
+      body: '{ "eula_accepted": true, "manifest": "{{ manifest_file.content }}" }'
```

Comment:
Out of interest, any special reason why you don't use a lookup plugin here to get rid of the previous step?

Reply:
Lookup plugins don't work with binary data.